core/server/helpers/proxy.js

```diff
@@ -40,6 +40,8 @@ module.exports = {
         isPrivacyDisabled: config.isPrivacyDisabled.bind(config)
     },
 
+    activeTheme: require('../services/themes/active'),
```

Comment:
IMO this results in unnecessary tight coupling between the theme service API and the image helper. Keeping in mind that helpers are being moved to Ghost-SDK where possible, not adding new couplings is desirable.

I think we should pass the `image_sizes` config to the helper using the same method we already use for `posts_per-page`, which can be found here: https://github.com/TryGhost/Ghost/blob/f2b44322a3883bdfef500e17ada7c2328d9915d4/core/server/services/themes/middleware.js#L73

Reply:
Note: @JohnONolan found a bug (of course) where uploading a theme without image sizes still loads image sizes.

Without having double checked the reproduction case or specifics, my first guess is that it happens when uploading a theme without sizes after one that does, and that the config is incorrectly cached between themes?

Hopefully switching out this code will kill 2 birds with 1 stone

Reply:
:grimacing: nice bug!